macOS/DuckDuckGo/Updates/UpdateController.swift

```diff
@@ -186,12 +213,33 @@ final class UpdateController: NSObject, UpdateControllerProtocol {
 
         Logger.updates.log("Checking for updates respecting rollout")
 
+        userDriver?.userCheckedForUpdates()
```

Comment:
This should be updated when the user checks for updates, immediately.

Reply:
Actually this should be inside `checkForUpdateSkippingRollout()` if you want it to happen when the user explicitly/force checks for updates.

Reply:
Updated the name for clarity.

The last update check date should be updated regardless of whether the check was user-initiated or not.  This is especially relevant for automatic updates.